testing/_py/test_local.py

```diff
@@ -9,9 +9,10 @@
 from unittest import mock
 import warnings
 
-from py import error
 from py.path import local
 
+from py import error
```

Comment:
This doesn't seem right. Why does ruff want inverse sort inhere?

Reply:
No idea TBH 🤷‍♂️ 

I don't care too much as long as it is consistent in different machines/computers.